sdk/storage/Azure.Storage.DataMovement/src/Shared/DataMovementConstants.cs

```diff
@@ -267,5 +270,10 @@ internal static class ErrorCode
             internal static readonly string[] CannotOverwrite = { "BlobAlreadyExists", "Cannot overwrite file." };
             internal static readonly string[] AccessDenied = { "AuthenticationFailed", "AuthorizationFailure", "access denied" };
         }
+
+        internal static class Share
```

Comment:
I think we should probably make a Share specific constants file in the Share package like I did for Blob? We will have a bunch of constants once checkpoint stuff comes in anyway.

Reply:
Sure I can make that